helm/nessie/values.yaml

```diff
@@ -119,6 +119,330 @@ bigtable:
     # # -- The secret key storing the Google Cloud service account JSON key.
     # key: sa_json
 
+# -- The Nessie catalog server configuration.
+catalog:
+
+  # -- Whether to enable the REST catalog service.
+  enabled: false
+
+  # -- Iceberg catalog settings.
+  iceberg:
```

Comment:
The general grouping is tentative, if you feel options could be grouped otherwise, I'm happy to change.